drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/util/Skipper.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.guided.dtable.backend.server.conversion.util;
+
+import java.util.Objects;
+
+import org.drools.workbench.models.guided.dtable.shared.model.AttributeCol52;
+import org.drools.workbench.models.guided.dtable.shared.model.BaseColumn;
+import org.drools.workbench.screens.guided.dtable.shared.XLSConversionResultMessage;
+import org.drools.workbench.screens.guided.dtable.shared.XLSConversionResultMessageType;
+
+public class Skipper {
+
+    private static final String DIALECT = "dialect";
+
+    private Skipper() {
+    }
+
+    public static boolean shouldSkip(final NotificationReporter notificationReporter,
+                                     final BaseColumn baseColumn) {
+        if (isDialect(baseColumn)) {
+            notificationReporter.report(new XLSConversionResultMessage(XLSConversionResultMessageType.DIALECT_NOT_CONVERTED,
+                                                                       "Dialect is not a supported column type in XLS Decision tables. Conversion ignored this column."));
+            return Objects.equals(((AttributeCol52) baseColumn).getAttribute(), "dialect");
+        } else {
+            return false;
+        }
+    }
```

Comment:
We break the rule of single responsibility. Method `shouldSkip` returns `boolean` what is straightforward. However it also store some message into `notificationReporter`. From my point of view that is bad pattern - not so intuitive. I propose to rewrite or at least add javadoc. What do you think?

Reply:
@jomarko This is true, but at this point I come up with anything better so added JavaDoc.